tests/OrganizationsTest.php

```diff
@@ -69,4 +69,137 @@ public function testOrganizationDetail()
             $this->assertEquals(404, $e->getCode());
         }
     }
+    
+    public function testUpdateOrganization()
+    {
+        $organization = $this->client->createOrganization($this->testMaintainerId, [
+            'name' => 'Test org',
+        ]);
+
+        $this->assertEquals("Test org", $organization['name']);
+        $this->assertEquals(1, (int) $organization['allowAutoJoin']);
+
+        $org = $this->client->updateOrganization($organization['id'], [
+            "name" => "new name",
+            "allowAutoJoin" => 0
+        ]);
+
+        $this->assertEquals("new name", $org['name']);
+        $this->assertEquals(0, (int) $org['allowAutoJoin']);
+    }
+
+    public function testOrganizationUsers()
+    {
+        $organization = $this->client->createOrganization($this->testMaintainerId, [
+            'name' => 'Test org',
+        ]);
+        $admins = $this->client->listOrganizationUsers($organization['id']);
+        $this->assertCount(1, $admins);
+
+        $this->client->addUserToOrganization($organization['id'], ['email' => 'spam@keboola.com']);
+
+        $admins = $this->client->listOrganizationUsers($organization['id']);
+        $this->assertCount(2, $admins);
+
+        $foundUser = null;
+        foreach ($admins as $user) {
+            if ($user['email'] == 'spam@keboola.com') {
+                $foundUser = $user;
+                break;
+            }
+        }
+        if (!$foundUser) {
+            $this->fail('User should be in list');
+        }
+
+        $this->client->removeUserFromOrganization($organization['id'], $foundUser['id']);
+
+        $admins = $this->client->listOrganizationUsers($organization['id']);
+        $this->assertCount(1, $admins);
+    }
+    
+    public function testOrganizationPermissions()
```

Comment:
would probably be better to split into multiple functions, each covering one case only. it's testing too many things at once and in this approach if any assertion fails the remaining tests are skipped, which could lead to confusion or harder debug.